dominion/dominion_ui/src/components/Search/SearchResults.js

```diff
@@ -50,7 +48,12 @@ const styles = theme => ({
   }
 });
 
+function Capitalize(strItem) {
+  return strItem[0].toUpperCase() + strItem.slice(1).toLowerCase();
+}
```

Comment:
My understanding is this `capitalize`-ation is only for presentation purposes only. Therefore we should use `css`. e.g.
```javascript
...
  textTransform: 'capitalize'
}
...
```

Reply:
@kilemensi I have made them lowercase then used textTransform.